props/saber_fett263_buttons.h

```diff
@@ -381,160 +1813,2794 @@ SaberFett263Buttons() : PropBase() {}
+        if (!menu_) {
+          current_menu_angle_ = fusor.angle2();
+          menu_ = true;
+          menu_type_ = MENU_VOLUME;
+          if (SFX_vmbegin) {
+            PlayMenuSound("vmbegin.wav");
+          } else {
+            PlayMenuSound("mvolume.wav");
+          }
+        }
+        return true;
+          
+      case EVENTID(BUTTON_AUX, EVENT_CLICK_LONG, MODE_OFF):
+        current_menu_angle_ = fusor.angle2();
+        PlayMenuSound("mpreset.wav");
+        menu_ = true;
+        menu_type_ = MENU_PRESET;
+        return true;
+
+      case EVENTID(BUTTON_AUX, EVENT_CLICK_SHORT, MODE_OFF):          
+        if (menu_) {        
+          switch (menu_type_) {
+            case MENU_REHEARSE:
+              PlayMenuSound("rehrsold.wav");
+              menu_ = false;
+              menu_type_ = MENU_TOP;
+              return true;
+            case MENU_PRESET:
+              PlayMenuSound("mselect.wav");
+              first_preset();
+              if (SFX_quote) {
+                SFX_quote.Select(-1);
+              }
+              menu_ = false;
+              menu_type_ = MENU_TOP;
+              wav_player.Free();
+              return true;
+              break;
+            case MENU_VOLUME:
+              if (SFX_vmend) {
+                PlayMenuSound("vmend.wav");
+              } else {
+                PlayMenuSound("mexit.wav");
+              }
+              menu_ = false;
+              menu_type_ = MENU_TOP;
+              wav_player.Free();
+              return true;
+              break;
+            case TRACK_PLAYER:
+              track_mode_ = PLAYBACK_RANDOM;
+              PlayMenuSound("mrandom.wav");
+              menu_ = false;
+              menu_type_ = MENU_TOP;
+              wav_player.Free();
+              return true;
+              break;
+          }
+        } else { 
+          if (fusor.angle1() < - M_PI / 4) {
+            previous_preset();
+          } else {
+            next_preset();
+          }
+          if (SFX_quote) {
+            SFX_quote.Select(-1);
+          }
+        }
+        return true;
+
+        // Handle double-click with preon
+      case EVENTID(BUTTON_POWER, EVENT_DOUBLE_CLICK, MODE_OFF):
+        if (on_pending_) {
+          if (SetMute(true)) {
+            unmute_on_deactivation_ = true;
+          }
+          return true;
+        }
+        return false;
+
+      case EVENTID(BUTTON_POWER, EVENT_DOUBLE_CLICK, MODE_ON):
+        if (millis() - activated_ < 500) {
+          if (SetMute(true)) {
+            unmute_on_deactivation_ = true;
+          }
+        }
+        return true;
+
+      case EVENTID(BUTTON_POWER, EVENT_RELEASED, MODE_OFF):
+        if (menu_ && menu_type_ == TRACK_PLAYER) {
+          menu_ = false;
+          menu_type_ = MENU_TOP;
+          return true;
+        } 
+        return false;
+
+      case EVENTID(BUTTON_AUX, EVENT_HELD_LONG, MODE_OFF | BUTTON_POWER):
+#ifdef FETT263_EDIT_MODE_MENU
```

Comment:
I'm guessing this is entering the menues?
A comment would help so I don't have to guess.

Reply:
comment added